plugins/cairo-lang-macro/src/lib.rs

```diff
@@ -9,6 +9,14 @@ use std::num::NonZeroU8;
 use std::slice;
 use std::vec::IntoIter;
 
+/// Free the memory allocated for the [`StableProcMacroResult`].
+///
+/// # Safety
+#[no_mangle]
+pub unsafe extern "C" fn free_result(result: StableProcMacroResult) {
+    ProcMacroResult::from_owned_stable(result);
+}
```

Comment:
I believe this should be `#[doc(hidden)]` as this is implementation detail which the user should not bother about.

Also please document here what we are actually doing with this `#[no_mangle]` thingy